### ISSUE TITLE
Bloodlord Weapon Gone

### ISSUE THREAD

**Issue by Itmritual** [WarEmu/WarBugs#16376]
I had the Bloodlord Weapon in my bag the last time I logged on, and I had it in there for a while. But today I logged in and it was just gone. I looked through all of my backpack and bank and it's just not there.  I thought maybe it just reset so I went back to the influence trainer to see if I could grab it again but no. 
![Eyztym_055](https://user-images.githubusercontent.com/31557076/79184913-8162b780-7de3-11ea-9022-6d5d8374b340.jpg)
![Eyztym_056](https://user-images.githubusercontent.com/31557076/79184933-8b84b600-7de3-11ea-90e3-aa64764351f9.jpg)
![Eyztym_057](https://user-images.githubusercontent.com/31557076/79184945-8fb0d380-7de3-11ea-9b05-e99c3432f3a6.jpg)

**Comment by Kaelang** [comment]
The team do not assist with item recoveries. That being said, I can't work out why this would have happened in the first place. Do you have any other information to provide so the team can investigate this to hopefully stop something along these lines happening again?

**Comment by nnnnNATHANnnnn** [comment]
It’s really unfortunate there’s no way the ROR team can make exceptions for item recoveries for items that can never ever be re farmed without litterally making an entire new character. Really sorry to see this happened to you.... on the bright side, new sc weapons and bloodlord doesn’t have a unique appearance :p 

I almost accidentally deleted my sentinel jewel last week, if I had somehow accidentally clicked yes I might have just quit the game or at the very least delete my toon and make a new sorc if I couldn’t get it replaced... you’re supposed to get two jewels from the influence to make this not be an issue but mine bugged and I only got one.

P.S this issue could be fixxed by making the influence “reset” after completion. Maybe keep a little note in the TOK saying you completed it once or whatever, but just have the entire thing reset so if needed players could just run the dungeons over again and re farm the inf items. I don’t think this would hurt anyone. I wouldn’t be really that sad if I hypothetically deleted my sentinel jewel and I knew I could just do another 13 runs and get it back.

**Comment by Itmritual** [comment]
The only thing I can think of is occasionally my backpack will not be working. I'll go onto the crafting tab and anything below the first two bars will be there but I won't be able to touch them. Like if I took something from one of the top two bars and dragged it to the ones below it would ask me if I wanted to delete the item. I would have to reload to make it work again. That's happened to me on occasion and I did have the Bloodlord weap in one of the three bottom bars. But keep in mind that I've only noticed this happen in the crafting part of my bag, next time I'll check and see if it also extends to the normal backpack.
